cli/src/pcluster/aws/fsx.py

```diff
@@ -17,17 +17,32 @@ class FSxClient(Boto3Client):
 
     def __init__(self):
         super().__init__("fsx")
+        self.cache = {}
+        Cache.add_cache(self.cache)
```

Comment:
`Cache.add_cache(self.cache)` ? :D

There is something we can improve in both variable names and methods.
1. let's use a more self-explicative name for the variable `file_systems_cache` to be clear what we're going to cache in this variable.
2. what about renaming `add_cache` to: `insert` ?

Reply:
Hahaha. Not valid anymore